src/components/network-modification-tree-pane.js

```diff
@@ -604,8 +598,8 @@ export const NetworkModificationTreePane = ({
 
                 <StudyDrawer
                     open={isModificationsDrawerOpen}
-                    drawerClassName={classes.nodeEditor}
-                    drawerShiftClassName={classes.nodeEditorShift}
+                    drawerStyle={styles.nodeEditor} // TODO Why does this have to be set in the parent, if StudyDrawer is only used here ? Let's remove the useless props and define the style in StudyDrawer
```

Comment:
Why not in this PR ?

Reply:
This is to have an easier to review PR. The next one is ready to push, I'm just waiting for this one to be merged.